src/Hl7.Fhir.Shims.STU3AndUp/FhirPath/FhirPathExtensions.cs

```diff
@@ -48,23 +48,23 @@ public static class FhirPathExtensions
 
         /// <inheritdoc cref="FhirPathCompilerCache.Select(ITypedElement, string, EvaluationContext?)"/>
         public static IEnumerable<Base?> Select(this Base input, string expression, FhirEvaluationContext? ctx = null)
-            => CACHE.Select(input.ToTypedElement(), expression, ctx ?? FhirEvaluationContext.CreateDefault()).ToFhirValues();
+            => CACHE.Select(input.ToTypedElement(), expression, ctx ?? new FhirEvaluationContext(input.ToTypedElement())).ToFhirValues();
```

Comment:
I think we should obsolete CreateDefault(), and/or create a new version of it that takes the current focus as a parameter.